j2k/src/org/jetbrains/kotlin/j2k/ExpressionConverter.kt

```diff
@@ -867,7 +912,8 @@ class DefaultExpressionConverter : JavaElementVisitor(), ExpressionConverter {
                 if (parameters.size == 1 && !isKotlinFunctionType) {
                     // for lambdas all parameters with types should be present
                     emptyList()
-                } else {
+                }
+                else {
```

Comment:
@piotrek1543 Please let the Kotlin team decide on the coding conventions used in the Kotlin project. Thanks.